src/goto-programs/remove_virtual_functions.h

```diff
@@ -65,4 +66,19 @@ void remove_virtual_function(
   const dispatch_table_entriest &dispatch_table,
   virtual_dispatch_fallback_actiont fallback_action);
 
+/// Given a function expression representing a virtual method of a class,
+/// the function computes all overridden methods of that virtual method.
+/// \param function: The virtual function expression for which the overridden
+///                  methods will be searched for.
+/// \param symbol_table: A symbol table.
+/// \param class_hierarchy: A class hierarchy.
+/// \param [out] overridden_functions: Output collection into which all
+///                                    overridden functions will be stored.
+void get_virtual_function_callees(
+  const exprt &function,
+  const symbol_tablet &symbol_table,
+  const class_hierarchyt &class_hierarchy,
+  dispatch_table_entriest &overridden_functions);
```

Comment:
I'd prefer that a `get_X` method had a return value. Otherwise, maybe `collect_X` is a better name.